src/components/shared/PrizesGallery.tsx

```
@@ -0,0 +1,476 @@
+        </div>
+
+        {/* Main Carousel Container */}
+        <div className="relative w-full max-w-7xl mx-auto">
+          {/* Navigation Buttons - only show if more than 1 prize */}
+          {filteredPrizes.length > 1 && (
+            <>
+              <button
+                onClick={prevSlide}
+                disabled={isTransitioning}
+                className="absolute left-2 sm:left-4 top-1/2 -translate-y-1/2 z-20 w-12 h-12 sm:w-16 sm:h-16 bg-theme-secondary/70 backdrop-blur-xl minimal-radius flex items-center justify-center text-theme-foreground hover:bg-theme-accent transition-all duration-500 ease-in-out border border-theme-border shadow-2xl theme-transition hover:scale-105 focus-ring disabled:opacity-50"
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)